shared/styles/index.desktop.js

```diff
@@ -107,7 +107,7 @@ export const backgroundURL = (...to: Array<string>) => {
 export const hairlineWidth = 1
 export const styleSheetCreate = (obj: Object) => obj
 export const collapseStyles = (styles: Array<Object> | Object) => {
-  if (isArray) {
```

Comment:
was this the fix for the root cause for the `vUsernameContainerStyle` thing above?

Reply:
No this is unrelated, I ran into an exception with this function trying to spread a primitive and noticed this wasn't checking correctly.